src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -255,6 +266,45 @@ public void importFromVaporAndConcreteReferences() {
             "import java.util.List;\n\n"));
   }
 
+  @Test
+  public void writeAnonymousClassExprImports(){
+    ConcreteReference stringListRef = ConcreteReference.builder().setClazz(List.class).setGenerics(Arrays.asList(ConcreteReference.withClazz(String.class))).build();    
```

Comment:
Can we ensure the formatter picks up this file as well?

Also, let's try using a class that is not String and needs to be imported, so we can see that the import writer actually picks up the references as well.

Reply:
sent a fix PR to include the writer files when running format tools. Thanks! And also used `IOException` and `MethodDefinition` to test import.